azure-iot-device/tests/common/test_mqtt_transport.py

```diff
@@ -94,7 +180,7 @@ def test_operation_infrastructure_set_up(self, mocker):
         assert transport._op_manager._unknown_operation_completions == {}
 
 
-@pytest.mark.describe("MQTTTransport - .connect()")
+@pytest.mark.describe("MQTTTransport - .connect() / Event: Connected, ConnectionFailure")
```

Comment:
>Connected [](start = 59, length = 9)

Is "Connected" really an externally driven event? How can connect be externally driven? Isn't "ConnectionFailure" the only externally driven event?

Reply:
Neither _should_ be externally driven.  

---
In reply to: [307910811](https://github.com/Azure/azure-iot-sdk-python-preview/pull/136#discussion_r307910811) [](ancestors = 307910811)